.vscode/quarto.code-snippets

```diff
@@ -0,0 +1,139 @@
+{
+	// Place your snippets for Quarto here. 
+	// Each snippet is defined under a snippet name and has a prefix, body and description. 
+	// The prefix is what is used to trigger the snippet and the body will be expanded and inserted. 
+	// Possible variables are:
+	//
+	// - $1 and $2 for tab stops, 
+	// - $0 for the final cursor position, and 
+	// - ${1:label}, ${2:another} for placeholders
+	// 
+	// Placeholders with the same ids are connected.
+	//
+	// Don't forget to edit VS Code to accept tab completion for Quarto - this is not available through the standard settings for Quarto
+	// Go to VS Code settings (Click gear icon in bottom left corner or use the shortcut Cmd+,) and search for: @lang:quarto snippets
+	// Set "Tab Completion" to "On"
+	// Ensure that "Show Snippets" is ticked
+	// Optional: Choose where the snippets comes in the list of suggestions in the field "Editor: Snippet Suggestions"
+	// 
+	// How to use the code snippets:
+	// After you have set the above settings, a suggestion for one of the code snippets below will pop up automatically, when
+	// you start writing the prefix. Press "Enter" or "Tab" to insert the code snippet.
+
+	"Insert TODO formatting":{
+		"prefix": "TODO",
+		"body": [
+			"<!--TODO ${0:Write text here}-->"
+		],
+		"description": "Insert TODO formatting"
+	}
+	
+	"Insert bash formatted text":{
+		"prefix": "bash",
+		"body": [
+			"``` bash"
+			"${0:Write text here}"
+			"```"
+		],
+		"description": "Insert bash formatted text"
+	}
+	
+	"Insert title section":{
+		"prefix": "ttl_sec",
+		"body": [
+			"---"
+			"title: \"\""
+			"description: \"Our reasons for ...\""
+			"author: \"\""
+			"date: last-modified"
+			"categories:"
+			"---"
+		],
+		"description": "Insert title section"
+	}
+
+	"Insert a hidden comment section":{
+		"prefix": "hddn",
```

Comment:
I think it's fine to use full words for these. So `hidden` works fine. I've also already changed this and will push up.